lib/dnsimple/client/vanity_name_servers.rb

```diff
@@ -0,0 +1,43 @@
+module Dnsimple
+  class Client
+    module VanityNameServers
+
+      # Enable vanity name servers for a domain. 
+      #
+      # @see https://developer.dnsimple.com/v1/nameservers/vanity-nameservers/#enable
+      #
+      # @param  [#to_s] domain The domain id or domain name.
+      # @param  [Hash] names A hash of up to 4 external name servers; hash keys
+      # are ns1 through ns4, e.g.
+      #         {
+      #           "ns1" => "ns1.example.com",
+      #           "ns2" => "ns2.example.com"
+      #         }
+      #
+      # @return [void]
+      # @raise  [RequestError] When the request fails.
+      def enable_vanity_name_servers(domain, names)
```

Comment:
I just noticed the method is missing the `option = {}` final parameter. See for example
https://github.com/aetrion/dnsimple-ruby/blob/master/lib/dnsimple/client/domains_privacy.rb#L13

The same applies to disable.

The reason it's there, is because it allows to pass custom headers and params to the client.
See https://github.com/aetrion/dnsimple-ruby/commit/eb452f9b3c3e98b48784fb6820884ea43986cd49